analysis/study_definition.py

```diff
@@ -121,20 +321,81 @@
         },
     ),
     rural_urban=patients.address_as_of(
```

Comment:
not in protocol - delete?

Reply:
IMD is in the protocol. We will remove rural_urban.